client/src/components/home.tsx

```diff
@@ -14,6 +16,10 @@ import { selectSourceMap, ViewType } from './sourceMapSlice'
 const home  = () => {
 
   const sourceMap = useAppSelector(selectSourceMap)
+
+  const spanList = useAppSelector(selectSpanMap)
```

Comment:
Should these variables be the same name? AKA SpanList != SpanMap?

Reply:
spanList != spanMap